features/p2/poms/pom.xml

```diff
@@ -42,7 +42,7 @@
     <groovy.eclipse.compiler.version>2.9.2-01</groovy.eclipse.compiler.version>
     <groovy.eclipse.batch.version>2.4.3-01</groovy.eclipse.batch.version>
 
-    <ohdr.version>1.0.40</ohdr.version>
+    <ohdr.version>1.0.41</ohdr.version>
```

Comment:
Will merge as soon as @kaikreuzer informs us that ohdr 1.0.41 is published.

Reply:
That's right! I studied how it all works for creating my own P2 repo. ;-) As a sanity check you can also check that the new version is available at the [OH P2 bintray repo](https://bintray.com/openhab/p2/openhab-deps-repo).

Reply:
It has automatically been published by running https://ci.openhab.org/job/openHAB-Deps-Repo/48/.